qa/L0_perf_deeprecommender/test.sh

```diff
@@ -177,7 +177,8 @@ rm -fr models && mkdir -p models && \
     cp -r $REPODIR/perf_model_store/deeprecommender_graphdef tfmodels/. && \
     (cd tfmodels/deeprecommender_graphdef && \
         sed -i "s/^max_batch_size:.*/max_batch_size: ${STATIC_BATCH}/" config.pbtxt && \
-        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
+        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt && \
```

Comment:
Why are these being handled differently that L0_perf_resnet? All instance_group, dynamic_batcher, etc. config file changes should happen in run_test.sh.

Reply:
Moved to run_test.sh